osu.Game/Beatmaps/ControlPoints/ControlPointInfo.cs

```diff
@@ -170,12 +170,12 @@ public void RemoveGroup(ControlPointGroup group)
         /// </summary>
         /// <param name="list">The list to search.</param>
         /// <param name="time">The time to find the control point at.</param>
-        /// <param name="prePoint">The control point to use when <paramref name="time"/> is before any control points. If null, a new control point will be constructed.</param>
+        /// <param name="fallback">The control point to use when <paramref name="time"/> is before any control points.</param>
         /// <returns>The active control point at <paramref name="time"/>, or a fallback <see cref="ControlPoint"/> if none found.</returns>
-        private T binarySearchWithFallback<T>(IReadOnlyList<T> list, double time, T prePoint = null)
-            where T : ControlPoint, new()
+        private T binarySearchWithFallback<T>(IReadOnlyList<T> list, double time, T fallback)
+            where T : ControlPoint
         {
-            return binarySearch(list, time) ?? prePoint ?? new T();
+            return binarySearch(list, time) ?? fallback;
```

Comment:
The removal of the `new T()` final-chance-fallback is causing the test failures here.